### PR TITLE
add feedback and getting started menu items

### PR DESCRIPTION
**Pull request by devoncarew**
- on the shiny new `Tools` menu, add `Getting Started` and `Submit Feedback` items
- add a new issue template for the repo

screen shot 2016-12-10 at 9 21 24 am

fix https://github.com/flutter/flutter-intellij/issues/519, fix https://github.com/flutter/flutter-intellij/issues/352

@pq, @mit-mit

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+[github.com/flutter/flutter](https://github.com/flutter/flutter/issues/new).
+
+## Steps to Reproduce
+
+_Please tell us what you were doing and what went wrong_
+
+
+## Version info
+
+Please paste the output of running `flutter doctor` here. It will provide the version
```
Comment (pq) [inline ISSUE_TEMPLATE.md]: Should we point them to the menu action here (**Tools/Flutter/Flutter Doctor**)?

**Comment by pq** [comment]
This is awesome.

FWIW I'll follow-up with a PR for `flutter packages get` and `flutter packages upgrade` in a bit.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (devoncarew) [inline ISSUE_TEMPLATE.md]: will do

**Comment by pq** [comment]
I've got some other items to add once this lands too. 🎉

**Comment by mit-mit** [comment]
Nice :-)

**Pull request merged**